.github/workflows/linux.yml

```diff
@@ -0,0 +1,30 @@
+name: Linux
+
+on:
+  push:
+    branches:
+      - '*'
+    tags-ignore:
+      - '*'
+  pull_request:
+
+jobs:
+  raku:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
```

Comment:
Unless I'm wrong, the only difference between the two workflows is the operating system. Why don't you just add "macos" here? Even if there's a difference, you can use `if` in steps that depend on the value of the matrix that's being used.